en/gps_compass/rtk_gps.md

```diff
@@ -140,6 +157,70 @@ See the [EKF2 GPS Configuration](../advanced_config/tuning_the_ecl_ekf.md#gps) s
 -->
 
 
-## Vehicle Setup Example
+### Vehicle Setup Example
 
 The airframe build topic [DJI Flamewheel 450 with distance sensor and RTK GPS](../frames_multicopter/dji_flamewheel_450.md) describes an airframe setup with the Here+ RTK GPS and a Pixhawk 3 Pro.
+
+
+## RTK GPS Heading Setup/Configuration (U-blox F9P)
```

Comment:
Maybe this should better go as a separate page under `GPS/Compass`, since that is the actual use-case: you have 2 GPS antennas and get heading estimation, and the rest (e.g. RTK) is implementation/device specific.

Reply:
The whole point of this was to try join all config together :-). Sometimes when you try things it makes it obvious why you'd rather not.

OK. Update to split it back out. Now it is a sub-topic.